src/Type/Php/ClassExistsFunctionTypeSpecifyingExtension.php

```diff
@@ -34,6 +34,7 @@ public function isFunctionSupported(
 			'class_exists',
 			'interface_exists',
 			'trait_exists',
+			'enum_exists',
```

Comment:
alternatively we could move it into a separate extension, specialized on the enum case